docs/guidelines-for-reviewing-datascience-code.md

```diff
@@ -1,25 +1,33 @@
+# 10 Principles for Reviewing Data Science Code
 
-Code reviews are common practice in the software engineering field and have been for some time. They provide developers with a well defined approach to sharing knowledge, ensuring exposure to new ideas and technologies, enforcing quality assurance of code and standards for the engineering process across both the code base and the team itself.  As members of the AI CoE in the office of the CTO, we believe that having a well defined approach for how to do the same thing with data science and machine learning code is critical. Although, code is code, and we approach data science work as a subset of software engineering, there are some data science specific challenges associated with the code review process that should be addressed. Below are the 11 guidelines we use in our group to review data science work.     
+Code reviews are common practice in the software engineering field and have been for some time. They provide developers with a well defined approach to sharing knowledge, ensuring exposure to new ideas and technologies, enforcing quality assurance of code and standards for the engineering process across both the code base and the team itself. As members of the AI CoE in the office of the CTO, we believe that having a well defined approach for how to do the same thing with data science and machine learning code is critical. Although, code is code, and we approach data science work as a subset of software engineering, there are some data science specific challenges associated with the code review process that should be addressed. Below are the 10 principles we use in our group to review data science work.
 
-1. **Understand the PR** : When asked for a review, make sure to first understand the issue linked and the goal of the pull request.
+1. **Understand the Context** : Make sure you read the pull request description and any linked issues so you know the data scientist's goal. This will help you provide an appropriate level of feedback and focus on the meaningful changes. 
 
-2. **NB Review** : Use tools like NB review when possible while reviewing Jupyter Notebooks, this will make reviewing the notebooks easier.
+2. **Rich Diffs** : Use tools like [NBreview](https://www.reviewnb.com/) or [nbdime](https://github.com/jupyter/nbdime) when possible to review notebooks. These kinds of tools help you zero in on the notebook changes that actually matter. 
 
-3. **Don’t be afraid to ask questions** : When reviewing someone’s code, if you don't understand something or are curious about why a certain approach was implemented, ask questions and get clarification. 
+3. **Does it make sense?** : Notebooks generally contain a mix of code, markdown, plots. Do these elements reference each other correctly? Is the evaluation metric being used here suitable for the ML algorithm ? If you don’t understand something or are curious about why a certain approach was implemented, ask and get clarification.
```

Comment:
Small nit, and it could be just me, but "does it make sense" feels too broad. Could we maybe update this to "ensure coherence" or something along those lines?

Otherwise looks good! :smiley: